libs/design/src/molecules/callout/callout.component.scss

```diff
@@ -0,0 +1,152 @@
+@import '../../scss/daff-util';
+
+@mixin callout-headings($titleSize: 2.5rem, $bodySize: 1rem) {
+  &__title {
+    margin: 0;
+    padding: 0;
+    font-size: 1.5rem;
+
+    @include breakpoint(mobile) {
+      font-size: 2rem;
+    }
+
+    @include breakpoint(big-tablet) {
+      font-size: $titleSize;
+      line-height: 3rem;
+    }
+  }
+
+  &__body {
+    margin: 10px 0 0 0;
+    padding: 0;
+    font-size: $bodySize;
+    font-weight: 400;
+
+    @include breakpoint(big-tablet) {
+      margin: 25px 0 0 0;
+    }
+
+    + button,
+    + .daff-button-set {
+      margin-top: 50px;
+    }
+  }
+
+  &__title, &__body {
+    max-width: 1000px;
+    text-align: center;
+
+    @include breakpoint(mobile) {
+      text-align: left;
+    }
+  }
+}
+
+.daff-callout {
+  $root: &;
+  @include callout-headings();
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+
+  &__container {
```

Comment:
Can we re-use daff--container?

Reply:
Hrm, then maybe callout__container is a bad name? I think we should use the daff--container as much as possible.